src/parser/shaman/restoration/modules/talents/Downpour.tsx

```diff
@@ -35,15 +36,19 @@ class Downpour extends Analyzer {
   downpourHitsSum = 0;
   downpourTimestamp = 0;
 
-  constructor(...args) {
-    super(...args);
+  protected cooldownThroughputTracker!: CooldownThroughputTracker;
```

Comment:
Same comment as above, would be best if these were right below the deps declaration I think